city_scrapers/spiders/cook_medical_examiner.py

```diff
@@ -0,0 +1,100 @@
+        needs.
+        """
+        meetings = response.css(".field-items strong *::text").getall()
+        dates = []
+        for i, item in enumerate(meetings):
+            if "Time" in item:
+                times = re.findall("(1[0-2]|0?[1-9]):([0-5]\\d)\\s*([AaPp][Mm])", item)
+                start_time = times[0]
+                end_time = times[1]
+            if "Location" in item:
+                venue = item.split(": ")[1]
+                street = meetings[i + 1] + "; " + meetings[i + 2]
+                address = {
+                    "name": venue,
+                    "address": street.replace(u'\xa0', u' '),
+                    "coordinates": {
+                        'latitude': None,
+                        'longitude': None
+                    }
+                }
+            if re.match(
+                "((Mon|Tues|Wednes|Thurs|Fri|Satur|Sun)day), "
+                "((Jan|Febr)uary|March|April|May|June|July|August|"
+                "(Septem|Octo|Novem|Decem)ber) "
+                "[1-3]*\\d, "
+                "\\d{4}", item
+            ):
+                dates.append(item)
+
+        for i, d in enumerate(dates):
+            meeting = Meeting(
+                title=self._parse_title(response),
+                description=self._parse_description(i, response),
+                classification=ADVISORY_COMMITTEE,
+                start=self._parse_start(d, start_time),
+                end=self._parse_end(d, end_time),
+                all_day=False,
+                location=address,
+                links=self._parse_links(d),
+                source=self._parse_source(response),
+            )
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, response):
+        return response.css("#page-title *::text").get().strip().replace("\\", "")
+
+    def _parse_description(self, i, response):
+        """Parse or generate meeting description."""
+        if i == 0:
+            # We only have the agenda for the first meeting.
+            agenda = response.css("li:not([class*='leaf']) *::text").getall()
+            return "\n".join([a.strip().replace("\\", "") for a in agenda if a != ''])
+        else:
+            return ""
+
+    def _parse_start(self, date, start_time):
+        """Parse start datetime as a naive datetime object."""
+        if "Date" in date:
+            return
+        dt = datetime.datetime.strptime(date + str(start_time), "%A, %B %d, %Y('%I', '%M', '%p')")
+        return dt
+
+    def _parse_end(self, date, end_time):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        if "Date" in date:
+            return
+        dt = datetime.datetime.strptime(date + str(end_time), "%A, %B %d, %Y('%I', '%M', '%p')")
+        return dt
+
+    def _parse_links(self, item):
```

Comment:
If there aren't any links this should return an empty list instead of a list with one empty item. This site is an interesting case because the detail page is the agenda for the coming meeting.

I'd lean toward leaving this blank because the content of the page changes over time and each meeting doesn't get a unique URL so it would be obsolete after each new meeting, but if I'm missing something let me know!

Reply:
I can't find the detail page! :D Do you mean the agenda list? I have included that as a meeting description for the first meeting. Or is there a link to another agenda?

Reply:
I just meant the page you're currently scraping is the detail page, and I think your solution of including the agenda as the description works best, so returning an empty list here should be good

Reply:
Alright, thank you for your quick answers! I will leave it as it is and push the new version with the corrections.

Reply:
Sounds great! It looks like you’ve addressed all my feedback here though, by corrections do you just mean fixing the errors in the CI build? Looks like that’s just a style issue with yapf

Reply:
Yes, hopefully I've now fixed the styling issue as well :)